[Question] Using the beforeMount in js instead of ts

Hey there, I've tried to use your `beforeMount` function as mentioned in [this ticket](https://github.com/microsoft/playwright/issues/17704) to register global plugins for vue 2, but I am trying to use it an js instead of ts.

index.js
```javascript
import { beforeMount } from '@playwright/experimental-ct-vue2/hooks'; // <= "Resolve error: Cannot find module './node_modules/@playwright/experimental-ct-vue2/hooks.mjs'"
import { registerPlugins } from '@/register';

beforeMount(({ app }) => {
  registerPlugins(app);
});
```

Is that not possible yet or do I have to use it in another way?

--- a/packages/playwright-ct-vue2/.npmignore
+++ b/packages/playwright-ct-vue2/.npmignore
@@ -8,4 +8,4 @@
 !index.d.ts
 !index.js
 !hooks.d.ts
-!hooks.js
+!hooks.mjs

